Services/GEV/Registration/classes/class.gevWBDTPServiceRegistrationGUI.php

```diff
@@ -143,30 +128,44 @@ protected function noBWVId() {
 	}
 
 	protected function createBWVId() {
-		/*
-		if ($this->user_utils->canBeRegisteredAsTPService()) {
-			return $this->createTPServiceBWVId();
+		return $this->createTPBasisBWVId();
+	}
+
+	protected function createTPServiceBWVId() {
+		if($_POST["registration_type"] == "new") {
```

Comment:
Aus dem ersten Schritt, wo er auswählt ob neues konto oder bestehendes.

Am 24.11.15 um 15:45 schrieb Richard Klees:
>
> In 
> Services/GEV/Registration/classes/class.gevWBDTPServiceRegistrationGUI.php 
> <https://github.com/conceptsandtraining/ILIAS/pull/72#discussion_r45743393>:
>
> > @@ -143,30 +128,44 @@ protected function noBWVId() {
> >  	}
> >
> >  	protected function createBWVId() {
> > -		/*
> > -		if ($this->user_utils->canBeRegisteredAsTPService()) {
> > -			return $this->createTPServiceBWVId();
> > +		return $this->createTPBasisBWVId();
> > +	}
> > +
> > +	protected function createTPServiceBWVId() {
> > +		if($_POST["registration_type"] == "new") {
>
> Woher kommt der registration_type?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/conceptsandtraining/ILIAS/pull/72/files#r45743393>.
>

-- 
Stefan Hecken
Developer

---------------------------------------------
CaT Concepts and Training GmbH
Vorgebirgstraße 338
50969 Köln

Fon:   +49 (0)221 / 46 75 76 - 57
Mobil: +49 (0)178 / 49 04 598
Fax:   +49 (0)221 / 46 75 76 - 09

Mail: stefan.hecken@concepts-and-training.de
Web:  http://www.concepts-and-training.de

---------------------------------------------
Geschäftsführung:

Sven Kapust, Gerald Konrad, Alexandra Oehlke,
Sandra Röbbelen, Volker Röbbelen, Denis Witt

Amtsgericht Köln HRB 57804
Ust-ID-Nr.: DE 814694228
Sitz: Köln
---------------------------------------------